src/main/java/ch/njol/skript/ScriptLoader.java

```diff
@@ -1105,6 +1107,30 @@ public String run(final Matcher m) {
 					hasDelayBefore = hadDelayBeforeLastIf;
 					((Conditional) items.get(items.size() - 1)).loadElseIf(cond, (SectionNode) n);
 					hasDelayBefore = hadDelayBeforeLastIf.or(hadDelayAfterLastIf.and(hasDelayBefore.and(Kleenean.UNKNOWN)));
+				} else if (StringUtils.startsWithIgnoreCase(name, "parse if ")) {
+					name = name.substring("parse if ".length());
+					final Condition cond = Condition.parse(name, "can't understand this condition: '" + name + "'");
+					if (cond == null)
+						continue;
+					if (!(cond instanceof SkippableCondition)) {

```

Comment:
this is going to prevent non-skippableconditions from being used which is not ideal

Reply:
As discussed on discord, I removed this requirement.